src/Contract/Entity/TimestampableInterface.php

```diff
@@ -14,7 +14,7 @@ public function getCreatedAt(): DateTimeInterface;
 
     public function setCreatedAt(DateTimeInterface $createdAt): void;
 
-    public function getUpdatedAt(): DateTimeInterface;
+    public function getUpdatedAt(): ?DateTimeInterface;
 
-    public function setUpdatedAt(DateTimeInterface $updatedAt): void;
+    public function setUpdatedAt(?DateTimeInterface $updatedAt): void;
```

Comment:
Why this change?

Reply:
Just for consistency of the API. Doesn't make sense as I wrote in the description. I can remove that again, but the whole PR is just a suboptimal workaround, see my new comment in the issue #496